pocs/tests/test_images.py

```diff
@@ -132,14 +112,16 @@ def test_pointing_error(solved_fits_file):
     assert (perr.magnitude.to(u.degree).value - 1.9445870862060288) < 1e-5
 
 
-# def test_compute_offset_arcsec(solved_fits_file, unsolved_fits_file):
-#     img0 = Image(solved_fits_file)
-#     img1 = Image(unsolved_fits_file)
+def test_compute_offset_arcsec(solved_fits_file, unsolved_fits_file):
+    img0 = Image(solved_fits_file)
+    img1 = Image(solved_fits_file)
```

Comment:
We should probably have two FITS files in a sequence that we can use that actually generates an offset. I thought that is what the `unsolved_fits_file` was and at one point I think creating an `Image` would solve by default, but I'm wondering if this was commented out because that doesn't happen any more (by default). We could use `unsolved_fits_file` as `img1` and do an `img1.solve_field()` and then have an offset, but that might take more time then we want. Maybe could just check the offset in a test that is already solving?